proposed/http-message.md

```diff
@@ -1100,7 +1106,8 @@ interface StreamableInterface
      *
      * @see seek()
      * @link http://www.php.net/manual/en/function.fseek.php
-     * @return bool Returns TRUE on success or FALSE on failure.
+     * @return bool Returns TRUE on success.
```

Comment:
I don't understand why the boolean is needed if the success case is not throwing an exception.